Apps/Examples/Examples/Models/Examples.swift

```diff
@@ -199,22 +199,22 @@ public struct Examples {
                 description: "Add a custom rendered Metal layer.",
                 type: CustomLayerExample.self),
         Example(title: "Add a line with a color gradient",
-                description: "Add a line with a rainbow color gradient.",
+                description: "Load a polyline to a style using GeoJsonSource, display it on a map using LineLayer, and style it with a rainbow color gradient.",
```

Comment:
GeoJSONSource is capitalized according to [Swift naming conventions](https://swift.org/documentation/api-design-guidelines/#follow-case-conventions) rather than Java/JavaScript naming conventions.

```suggestion
                description: "Load a polyline to a style using GeoJSONSource, display it on a map using LineLayer, and style it with a rainbow color gradient.",
```